tests/goth/pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[build-system]
+requires = ["poetry_core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "yajsapi-integration-tests"
+version = "0.1.0"
+description = "Integration tests for yajsapi"
+authors = ["GolemFactory <contact@golem.network>"]
+license = "LGPL-3.0-or-later"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Framework :: AsyncIO",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Topic :: System :: Distributed Computing"
+]
+repository = "https://github.com/golemfactory/yajsapi"
+documentation = "https://handbook.golem.network/yajsapi/"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.8.0"
+pytest = "^5.4.3"
+pytest-asyncio = "^0.12.0"
+goth = {git = "https://github.com/golemfactory/goth.git", branch = "master"}
```

Comment:
Meanwhile `goth` got published on `pypi` so we can just say `goth = "^0.1.0"` here.

Reply:
Done.